NebulaModel/MultiplayerOptions.cs

```diff
@@ -129,6 +129,10 @@ public bool StreamerMode
     [Description("Keyboard shortcut to toggle the chat window")]
     public KeyboardShortcut ChatHotkey { get; set; } = new(KeyCode.BackQuote, KeyCode.LeftAlt);
 
+    [DisplayName("Player List Hotkey")]
+    [Description("Keyboard shortcut to display the Connected Players Window")]
+    public KeyboardShortcut PlayerListHotkey { get; set; } = new(KeyCode.Tab);
```

Comment:
No sure if I'm a fan of setting default key as `Tab`. The key is used in many places in default keybinds.
It would be quite annoying if the window kept popping out when doing other operations.

Reply:
- Changed to F1.
- No longer shows if escScreen, Techtree or dysonEditor windows are open